nifi-nar-bundles/nifi-standard-services/nifi-key-service-bundle/nifi-key-service/src/test/java/org/apache/nifi/key/service/StandardPrivateKeyServiceTest.java

```diff
@@ -118,6 +134,21 @@ void testGetPrivateKeyEncryptedKey() throws Exception {
         assertEquals(generatedPrivateKey, privateKey);
     }
 
+    private static String[] encryptionAlgorithms() {
+        return new String[] {
+                AES_128_CBC,
+                AES_192_CBC,
+                AES_256_CBC,
+                DES3_CBC,
+                PBE_SHA1_RC4_128,
+                PBE_SHA1_RC4_40,
+                PBE_SHA1_3DES,
+                PBE_SHA1_2DES,
+                PBE_SHA1_RC2_128,
+                PBE_SHA1_RC2_40
```

Comment:
It is not necessary to test all of these algorithms, particularly those algorithms where the only difference is the key size. I recommend a much smaller list:
```suggestion
                AES_256_CBC,
                DES3_CBC
```

Reply:
Thanks @exceptionfactory for your review!

Yes, it makes sense to narrow down the encryption algorithms to be tested.
However, I would keep one more: `PBE_SHA1_3DES`
This was the original test input and it ends up in the [PKCS12](https://github.com/bcgit/bc-java/blob/7349980f39aa26541f69d99e66416e84b9f7c005/pkix/src/main/java/org/bouncycastle/openssl/jcajce/JceOpenSSLPKCS8DecryptorProviderBuilder.java#L101) branch, so it is handled differently than `AES_256_CBC` and `DES3_CBC`.